getting_started/kubernetes-overview.adoc

```diff
@@ -0,0 +1,105 @@
+image::247-OpenShift-Kubernetes-Overview.png[]
+
+To solve this problem, you can use containerization technologies that segregate applications in a containerized environment. Similar to a VM, a container has its own filesystem, vCPU, memory, process space, dependencies, and more. Containers are decoupled from the underlying infrastructure, and are portable across clouds and OS distributions. Containers are inherently much lighter than a fully-featured OS, and are light weight isolated processes that run on the operating system kernel. VMs are slower to boot, and are an abstraction of physical hardware. VMs run on a single machine with the help of Hypervisor.
+
+You can perform the following actions by using Kubernetes:
+
+* Sharing resources.
+* Orchestrating containers across multiple hosts.
+* Installing new hardware configurations.
+* Running health checks and self-healing applications.
+* Scaling containerized applications.
+
+.Kubernetes Components
+[cols="1,2",options="header"]
+|===
+|Component |Purpose
+
+|`kube-proxy`
+|Runs on every node in the cluster and maintains the network traffic between the Kubernetes resources.
+
+|`kube-controller-manager`
+|Governs the state of the cluster.
+
+|`scheduler`
+|Allocates pods to nodes.
+
+|`etcd`
+|Stores cluster data.
+
+|`kube-api-server`
+|Validates and configures data for the API objects.
+
+|`kubelet`
+|Runs on nodes and reads the container manifests. Ensures that the defined containers have started and are running.
+
+|`kubectl`
+|Allows you to define how you want to run workloads. Use the `kubectl` command to interact with the `kube-api-server`.
+
+|Node
+|Node is a physical or VM in a Kubernetes cluster. The control plane manages every node and schedules pods across the nodes in the Kubernetes cluster.
+
+|`container-runtime`
+|`container-runtime` runs containers on a host operating system. You must install container-runtime on each node so that pods can run on the node.
+
+|Persistent storage
+|Stores the data even after the device is shut  off. Kubernetes uses persistent volumes to store the application data.
+
+|`container-registry`
+|Stores and accesses the container images.
+
+|Pod
+|The pod is the smallest logical unit in Kubernetes. A pod contains one or more containers to run in a worker node.
+|===
+
+A cluster is a single computational unit consisting of multiple nodes in a cloud environment. A Kubernetes cluster includes a control plane and worker nodes. You can run Kubernetes containers across various machines and environments. The control plane node controls and maintains the state of a cluster. You can run the Kubernetes application by using worker nodes. You can use the Kubernetes namespace to differentiate cluster resources in a cluster. Namespace scoping is applicable for resource objects, such as deployment, service, and pods. You can not use namespace for cluster-wide resource objects such as `storageclass`, nodes, and persistent volumes.
```

Comment:
Per [OpenShift doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references), When referring to API object terms in general usage, use lowercase and separate multi-word API objects.
```suggestion
A cluster is a single computational unit consisting of multiple nodes in a cloud environment. A Kubernetes cluster includes a control plane and worker nodes. You can run Kubernetes containers across various machines and environments. The control plane node controls and maintains the state of a cluster. You can run the Kubernetes application by using worker nodes. You can use the Kubernetes namespace to differentiate cluster resources in a cluster. Namespace scoping is applicable for resource objects, such as deployment, service, and pods. You cannot use namespace for cluster-wide resource objects such as storage class, nodes, and persistent volumes.
```